### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by 3TUSK**
<s>Please do not merge this one *temporarily*.</s>
Summon @bakaxyf @CannonFotter @Amamiya-Nagisa and @Joccob to help review. I did review for couple of times but there are serval entires which I am not sure. Also I am not sure if those guys have different opinion(s).
Also, from the code I discovered that those special revolvers *are* visible in the future... then I decided to start to translate those entries. At the same time those who are working on French translation also translated some of them. What do you think about it?
Edit: Good to go.

### CODE REVIEW

File: src/main/resources/assets/immersiveengineering/lang/zh_CN.lang [hunk 1]
```diff
@@ -436,12 +528,19 @@ ie.manual.entry.blastfurnaceBlock=搭建一个高炉,你需要用这27个方块
```
Comment (bakaxyf) [inline src/main/resources/assets/immersiveengineering/lang/zh_CN.lang]: https://github.com/bakaxyf/ImmersiveEngineering/commit/a625edd873e93936aefd9d380c8a31c9c84f4ab5#diff-c6dfd06119fbfb2ad8bb3d15898a95d9L389

File: src/main/resources/assets/immersiveengineering/lang/zh_CN.lang [hunk 2]
```diff
@@ -436,12 +528,19 @@ ie.manual.entry.blastfurnaceBlock=搭建一个高炉,你需要用这27个方块
 
 ie.manual.entry.steelconstruction.name=钢结构
 ie.manual.entry.steelconstruction.subtext=铆钉
-ie.manual.entry.steelconstruction0=钢不仅是高压设备不可或缺的一部分,也是非常有用,极好的建筑材料,钢制脚手架可用于快速搭建建筑结构,钢围栏能让你建立防线抵御敌人,钢制臂架构是你搭建高压继电器的依附点.
-ie.manual.entry.steelconstruction1=钢制接线器的作用与普通接线器类似,但它们是纯装饰性物品,而非用于传输能量.<br>它们无法接收普通线缆,但能够与麻线圈或钢线圈连接(见下页).<br>你可以手持工程锤右击来旋转它们.
+ie.manual.entry.metalconstruction0=除了防腐木，加工过的金属也可用于建筑。你可以将绝大多数金属制成块或台阶，你还可以用锤子把铁锭敲成sheetmetal block.
+ie.manual.entry.steelconstruction1=钢不仅是高压设备不可或缺的一部分,也是非常有用,极好的建筑材料,钢制脚手架可用于快速搭建建筑结构,钢围栏能让你建立防线抵御敌人,钢制臂架构是你搭建高压继电器的依附点.
+ie.manual.entry.steelconstruction2=钢制接线器的作用与普通接线器类似,但它们是纯装饰性物品,而非用于传输能量.<br>它们无法接收普通线缆,但能够与麻线圈或钢线圈连接(见下页).<br>你可以手持工程锤右击来旋转它们.
```
Comment (bakaxyf) [inline src/main/resources/assets/immersiveengineering/lang/zh_CN.lang]: metalconstruction https://github.com/BluSunrize/ImmersiveEngineering/commit/c38bd8fd9d023898cbe6c6446ea0334f788e8a9a#diff-5f481613904e64725ae03b2edaa59293L467

File: src/main/resources/assets/immersiveengineering/lang/zh_CN.lang [hunk 3]
```diff
@@ -223,25 +241,28 @@ itemGroup.ImmersiveEngineering=Immersive Engineering
...
 desc.ImmersiveEngineering.info.blockSide.NORTH=北面
 desc.ImmersiveEngineering.info.blockSide.SOUTH=南面
 desc.ImmersiveEngineering.info.blockSide.EAST=东面
 desc.ImmersiveEngineering.info.blockSide.WEST=西面
+desc.ImmersiveEngineering.info.connectEnergy.-1=未连接
+desc.ImmersiveEngineering.info.connectEnergy.0=能量输入
+desc.ImmersiveEngineering.info.connectEnergy.1=能量输出
+desc.ImmersiveEngineering.info.connectFluid.-1=未连接
+desc.ImmersiveEngineering.info.connectFluid.0=流体输入
+desc.ImmersiveEngineering.info.connectFluid.1=流体输出
```
Comment (bakaxyf) [inline src/main/resources/assets/immersiveengineering/lang/zh_CN.lang]: desc.ImmersiveEngineering.info.blockSide
https://github.com/BluSunrize/ImmersiveEngineering/blob/master/src/main/resources/assets/immersiveengineering/lang/en_US.lang#L257-L262

**Comment by bakaxyf** [comment]
https://github.com/BluSunrize/ImmersiveEngineering/commit/181978a46b230d2a95698f81451a1d0a0da5d685#diff-5f481613904e64725ae03b2edaa59293R436

File: src/main/resources/assets/immersiveengineering/lang/zh_CN.lang [hunk 4]
```diff
@@ -175,15 +189,16 @@ item.ImmersiveEngineering.tool.manual.name=工程师手册
 item.ImmersiveEngineering.revolver.normal.name=左轮手枪
```
Comment (bakaxyf) [inline src/main/resources/assets/immersiveengineering/lang/zh_CN.lang]: https://github.com/BluSunrize/ImmersiveEngineering/commit/9f01ac3e2c4b9703d597e1ca8cf07fda324ccf4f#diff-5f481613904e64725ae03b2edaa59293R179

**Comment by BluSunrize** [comment]
Tell me when to merge, alright?

**Comment by 3TUSK** [comment]
I will finish it today (EST)

**Comment by 3TUSK** [comment]
Alright I have done. I *am* sorry for the inconvenience.

**Comment by BluSunrize** [comment]
Not a problem :D

**Pull request merged**